mkdocs.yml

```diff
@@ -55,26 +45,53 @@ markdown_extensions:
   - pymdownx.superfences
   - pymdownx.keys
   - pymdownx.tabbed
+  - pymdownx.details
+  - pymdownx.tasklist
   - footnotes
   - def_list
-  - pymdownx.tasklist
   - meta
 
 plugins:
   - search
-  - localsearch
```

Comment:
@justasojourner Why was this removed?

Reply:
@wsoyinka This too -- surely removing search isn't wanted

Reply:
This is **localsearch** (on the local computer) not website search. Search works fine without it, in fact one of the reasons I removed it was because looking at dev window in production site it can be seen that the website cannot locate the relevant JavaScript. So it fails / does not load.